packages/hardware-trezor/src/transformers/keyPaths.ts

```diff
@@ -15,13 +15,13 @@ export const resolvePaymentKeyPathForTxIn = (
   }
 };
 
-// Resolves the stake key path for known addresses for the given reward address.
 export const resolveStakeKeyPath = (
   rewardAddress: Cardano.RewardAddress | undefined,
-  context: TrezorTxTransformerContext
+  context?: TrezorTxTransformerContext
```

Comment:
It is not possible to resolve stake key path without context, so this change seems a little bit odd

Reply:
you are right, nice catch. Thanks @mkazlauskas

Reply:
@marcin-mazurek done in [bda86db](https://github.com/input-output-hk/cardano-js-sdk/pull/1096/commits/bda86dbd93a5e526a3c4d4960052ba0506583219)

p.s. Will squash it once you approve the changes